src/style.css

```diff
@@ -50,18 +43,15 @@
   text-shadow: 0 3px 3px rgba(61, 78, 97, 0.3);
 }
 
-.form-wrapper:hover {
-  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
-}
-
 .search-image {
   object-fit: none;
 }
 
-label {
+.label-image {
```

Comment:
Why not just name `.label` or `.search-label`? It's a more generic name in this case and it points to inputs not strictly to `<img />` tag :)